packages/dm-core/src/styles/main.css

```diff
@@ -18,6 +18,34 @@ body {
   box-sizing: inherit;
 }
 
+#root {
+  display: flex;
+  height: 100%;
+  width: 100%;
+}
+
+.wrapper {
```

Comment:
How about we name this something else? 

since wrapper, and dm-plugin-wrapper are two very similar names, and don't actually convey much meaning. 

ie. flexColumnContainer

or for dm-plugin-wrapper something like 'plugin-padding'

Reply:
.flex-layout-container ? 🙂 
I think the dm-plugin-wrapper is fine because it can be used to target plugin-wrappers inside anything you want. So it's not necessarily just for padding

Reply:
.flex-layout-container: thats fine! 

About plugin-wrapper
Alright, if we are to add more things to it that makes sense. 
But then I propose we rename it at that point, and have a descriptive name for what it is now. Which is just the padding part of it. 
Just by reading the dm-pluigin-wrapper it's not very descriptive of what it is. However, if it is a many-thing container then it makes sense.